src/types/ErrorOption.ts

```diff
@@ -0,0 +1,5 @@
+export enum ErrorOption {
```

Comment:
Consider renaming the enum 'ErrorOption' to 'ErrorMessage' to better reflect its purpose.